fst-levenshtein/src/lib.rs

```diff
@@ -142,16 +143,20 @@ impl DynamicLevenshtein {
 impl Automaton for Levenshtein {
     type State = Option<usize>;
 
+    #[inline]
```

Comment:
Sorry, maybe I'm missing something here, but are methods in trait impls not considered generic? Do these need the inline annotations? (There are many more of them in this PR.)

Reply:
Methods in trait impls are not generic unless they are themselves generic function or the impl is generic, I'm pretty much sure. You can see the symbols listed in the comment above, many of them are methods in trait impls.